src/components/EditSections/EditUserInfo/EditUserInfo.js

```diff
@@ -71,112 +142,166 @@ class EditUserInfo extends React.Component {
       }
     ];
 
+    const offset = this.getPatronGroupOffset();
+    const group = _.get(this.props.patronGroups.find(i => i.id === this.props.selectedPatronGroup), 'group', '');
+    const footer = (
```

Comment:
Please rename this constant to e.g. `modalFooter`. Then it is clearer that this is the footer of the modal.